Added info about We did not receive the submission (#11534)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/office-365-security/submissions-error-messages.md

- You tried to submit an email message that was filtered by EOP or Defender for Office 365, but we're still in the process of collecting the required metadata (descriptive data) about the message.

  If you wait "a while" and submit the message again, the submission will be successful.
  
  
## We did not receive the submission, please fix the problem and resubmit

> If you encounter this error message, then either of the following conditions have occured:

Reviewer: ```suggestion
If you encounter this error message, then either of the following conditions has occurred:
```

> - You are trying to submit an email which has been deleted or is no longer in the mailbox or quarantine.

Reviewer: ```suggestion
- You are trying to submit an email that has been deleted or is no longer in the mailbox or quarantine.
```

> - You have Exchange Transport Rules (ETR) or Connectors or Data Loss Prevention (DLP) rules preventing the message from coming to us 

Reviewer: ```suggestion
- You have Exchange Transport Rules (ETR), connectors or Data Loss Prevention (DLP) rules preventing the message from reaching us. 
```

> So please make sure to check that both of these conditions are false before submitting the message again. 

Reviewer: ```suggestion
Be sure to check that both of these conditions are false before submitting the message again. 
```
